docs/pages/experiments/website/branding-theme-test.tsx

```diff
@@ -30,20 +32,26 @@ export default function BrandingThemeTest() {
           </Stack>
           <Stack direction="row" spacing={2} useFlexGap sx={{ width: 'fit-content', mt: 8 }}>
             <Button variant="contained" size="small" color="primary">
-              This button
+              Contained primary
             </Button>
             <Button variant="contained" size="small" color="secondary">
-              This button
+              Contained secondary
```

Comment:
I think there's room for improvement—or maybe it's a bug—with secondary contained button. In the light mode, it looks like a disabled button, and in the dark mode there's no contrast.

<img width="164" alt="Screenshot 2024-03-27 at 15 16 42" src="https://github.com/mui/material-ui/assets/37222944/8c47033e-ffbb-4662-be9a-1e6b55ad2c9e">
<img width="165" alt="Screenshot 2024-03-27 at 15 16 34" src="https://github.com/mui/material-ui/assets/37222944/59f638ef-bd18-47e9-a728-db130a368868">

Reply:
Haven't really added any custom styles for this one yet as we're not using it—just put it on the page so we could see them together. Will tweak it later on!